cli/jobs/dataprep/dask/src/startDask.py

```diff
@@ -0,0 +1,188 @@
+import os
+import argparse
+import time
+from dask.distributed import Client
+from azureml.core import Run
+import sys, uuid
+import threading
+import subprocess
+import socket
+
+from notebook.notebookapp import list_running_servers
+
+
+def flush(proc, proc_log):
+    while True:
+        proc_out = proc.stdout.readline()
+        if proc_out == "" and proc.poll() is not None:
+            proc_log.close()
+            break
+        elif proc_out:
+            sys.stdout.write(proc_out)
+            proc_log.write(proc_out)
+            proc_log.flush()
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--jupyter_token", default=uuid.uuid1().hex)
+    parser.add_argument("--script")
+
+    args, unparsed = parser.parse_known_args()
+
+    print(
+        "- env: AZ_BATCHAI_JOB_MASTER_NODE_IP: ",
```

Comment:
are these environment variables stable? will they persist with common runtime changes?

Reply:
I guess we will need to update once we shift to common runtime